easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb

```diff
@@ -0,0 +1,27 @@
+name = 'MUMPS'
+version = '5.3.5'
+versionsuffix = '-parmetis'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['http://mumps.enseeiht.fr/']
```

Comment:
Needs to be updated, sources are now at https://graal.ens-lyon.fr/MUMPS/ I think